format.js

```diff
@@ -61,6 +61,29 @@ function format_map(m) {
     return embed;
 }
 
+function format_online(players) {
+    players.sort((a, b) => {
+        if (a.rank_info.rank < b.rank_info.rank) return -1;
```

Comment:
You could just subtract them :)

Reply:
Resolved in https://github.com/arispoloway/tempus-discord-bot/pull/2/commits/63778ca1eccd7944977fd2ed403f5471db2dd379